src/ForumifyBundle.php

```diff
@@ -4,8 +4,33 @@
 
 namespace Forumify;
 
-use Forumify\Plugin\AbstractForumifyBundle;
+use Forumify\Plugin\AbstractForumifyPlugin;
+use Forumify\Plugin\PluginMetadata;
+use Forumify\Plugin\PluginPermissions;
 
-class ForumifyBundle extends AbstractForumifyBundle
+class ForumifyBundle extends AbstractForumifyPlugin
 {
+    public function getPluginMetadata(): PluginMetadata
+    {
+        return new PluginMetadata(
+            'Forumify Platform',
+            'Forumify',
+        );
+    }
+
+    public function getPluginPermissions(): PluginPermissions
+    {
+        $perms = [
+            'forumify' => [
+                'admin' => [
+                    'user' => ['view', 'modify', 'delete'],
```

Comment:
I prefer the term “edit” over “modify” so it’s consistent with the controller names